scripts/backport.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+
+# Original author: Tim Graham?.
+
+set -xue
+
+if [ -z $1 ]; then
+    echo "Full hash of commit to backport is required."
+    exit
+fi
+
+BRANCH_NAME=`git branch | sed -n '/\* stable\//s///p'`
+echo $BRANCH_NAME
+
+git reset --hard
+
+REV=$1
+: ${ORIGBRANCH=main}
+
+TMPFILE=tmplog.tmp
+
+# Cherry-pick the other commit
+#ORIGID=`git find-rev r${REV} ${ORIGBRANCH}`
+#if [ -z "${ORIGID}" ] ; then
+#    echo "Revision ${REV} not found in branch ${ORIGBRANCH}"
+#    exit 1
+#fi
```

Comment:
What's this?

Reply:
Good catch, it comes from the script in the wiki and I never paid attention to that part. Will remove it.